dbConfig.mjs

```diff
@@ -0,0 +1,12 @@
+import { Pool } from 'pg';
+
+const Pool = new Pool ({
+    user: "username",
+    host: "localhost",
+    database: "database_name",
+    password: "password",
+    port: 5432,
```

Comment:
It is not advisable to add sensitive information on a GitHub repository. All these information should be read from environment variables. Please read on `.env` files